pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -135,19 +135,37 @@ func (p *ProjectSpec) ObjClusterName() string {
 }
 
 // +genclient
-// +kubebuilder:skipversion
 // +genclient:nonNamespaced
+// +kubebuilder:resource:scope=Cluster
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// GlobalRole define global rules that can be applied to the local cluster and or every downstream cluster.
 type GlobalRole struct {
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+
+	// Standard object metadata; More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata.
+	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	DisplayName    string              `json:"displayName,omitempty" norman:"required"`
-	Description    string              `json:"description"`
-	Rules          []rbacv1.PolicyRule `json:"rules,omitempty"`
-	NewUserDefault bool                `json:"newUserDefault,omitempty" norman:"required"`
-	Builtin        bool                `json:"builtin" norman:"nocreate,noupdate"`
+	// DisplayName is the human-readable name displayed in the UI for this resource.
+	// +optional
+	DisplayName string `json:"displayName,omitempty" norman:"required"`
+
+	// Description holds text that describes the resource.
+	// +optional
+	Description string `json:"description"`
+
+	// Rules holds a list of PolicyRules that are applied to the local cluster only.
+	// +optional
+	Rules []rbacv1.PolicyRule `json:"rules,omitempty"`
+
+	// NewUserDefault if true all new users that are create are bound to this GlobalRole.
```

Comment:
```suggestion
	// NewUserDefault - if true, all new users are bound to this GlobalRole.
```

Reply:
Updated the comment. I didn't use a dash so that we have a complete sentence.
> As with packages (above) and funcs (below), doc comments for types start with complete sentences naming the declared symbol.

https://tip.golang.org/doc/comment